starter-code/src/app/components/my-movie/my-movie.component.html

```diff
@@ -0,0 +1,7 @@
+<p>
+  my-movie works!
+
+  this page is for {{movie.title}}
```

Comment:
Now we know that at the beginning `movie` it will be undefined. Don't forget to handle this error with a `*ngIf`